Dockerfile

```diff
@@ -15,7 +15,8 @@ ENV CONF_PHPFPM=/etc/php/7.0/fpm/php-fpm.conf \
     PHP_FPM_MEMORY_LIMIT=256M \
     PHP_FPM_MAX_EXECUTION_TIME=60 \
     PHP_FPM_UPLOAD_MAX_FILESIZE=1M \
-    NEWRELIC_VERSION=6.7.0.174
+    NEWRELIC_VERSION=6.7.0.174 \
+    CFG_APP_DEBUG=1
```

Comment:
Also, do we want to default to 1 or 0?  This will change default production behavior.  @bryanlatten